packages/studio/src/store/slices/createFileMetadataSlice.ts

```diff
@@ -1,82 +1,36 @@
 import {
   ComponentMetadata,
-  ComponentState,
   FileMetadata,
   FileMetadataKind,
-  ModuleMetadata,
 } from "@yext/studio-plugin";
 import initialStudioData from "virtual_yext-studio";
 import FileMetadataSlice from "../models/slices/FileMetadataSlice";
 import { SliceCreator } from "../models/utils";
-import removeTopLevelFragments from "../../utils/removeTopLevelFragments";
 
 const createFileMetadataSlice: SliceCreator<FileMetadataSlice> = (
   set,
   get
 ) => ({
-  UUIDToFileMetadata: removeTopLevelFragments(
-    initialStudioData.UUIDToFileMetadata
-  ),
+  UUIDToFileMetadata: initialStudioData.UUIDToFileMetadata,
   UUIDToImportedComponent: {},
   setFileMetadata: (metadataUUID: string, metadata: FileMetadata) =>
```

Comment:
do we still need this action? I don't think we should be updating file metadata through the UI anymore

Reply:
True, yeah in the past we only used it for modules and we no longer use it-- thanks for the catch!